jbehave-core/src/main/java/org/jbehave/core/reporters/JsonOutput.java

```diff
@@ -49,7 +49,7 @@
     private AtomicInteger subStepsLevel = new AtomicInteger();
     private final Map<Integer, ScenarioStatus> scenariosStatusPerLevels = new HashMap<>();
     private boolean stepPublishing = false;
-    private boolean examplesPublishing = false;
+    private Map<Integer, Boolean> examplesPublishingPerLevels = new HashMap<>();
```

Comment:
this field is very similar to `scenariosStatusPerLevels`, does it make sense to create a POJO that will contain `scenarioStatus` and `examplesPublish` fields?